### ISSUE TITLE
Various setup/readme metadata is incorrect

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7967]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7967|
|Type|enhancement|
|Created|2015-07-11 15:32:50Z|
|Branch|https://github.com/twisted/twisted/tree/fix-metadata-7967|

This needs to be cleaned up before release.

<details><summary>Searchable metadata</summary>

```
trac-id__7967 7967
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__ 
branch__branches_fix_metadata_7967 branches/fix-metadata-7967
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1436628770855844 1436628770855844
changetime__1436698183763562 1436698183763562
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [45229]) Branching to fix-metadata-7967.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Builders are looking okay. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek set owner to @hawkowl|
|-|-|

I would suggest to use consistently https in URIs if possible; zope.interface & win32 installers are currently on http.

Other than that LGTM.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45246]) Merge fix-metadata-7967: Fix setup/INSTALL metadata.

Author: hawkowl
Reviewer: hynek
Fixes: [#7967](https://github.com/twisted/twisted/issues/7967)
